src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -129,12 +129,16 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.5'">
     <PackageReference Include="System.Collections.Specialized" Version="4.0.1" />
     <PackageReference Include="System.Diagnostics.TextWriterTraceListener" Version="4.0.0" />
-    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.0.0" />
+    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
     <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
     <PackageReference Include="System.Net.Security" Version="4.3.2" />
     <PackageReference Include="System.Security.SecureString" Version="4.0.0" />
     <PackageReference Include="System.Threading.Thread" Version="4.3.0" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
```

Comment:
Do you really need this one in .netstandard2.0? TraceSource is part of the standard and should be in `netstandard.library\2.0.3\build\netstandard2.0\ref\netstandard.dll`

Reply:
yes, but this section has been added VS by default when I added this reference. So we followed this behavior for now

Reply:
The problem with using those 4.3.0 nugets in a netstandard2.0 target is that it references the old .net core 1.x runtime which brings in all kinds of nuget references from that runtime.

That means that anyone consuming that netstandard2.0 target in a .NET Core app 2 or 3 project would get those old nugets restored, which might cause problems.

Reply:
Sounds reasonable to me. And I don't see any disadvantages in this removing. So, I've applied it. 
Thanks.

Reply:
I agree that it is better to remove the dependency on the System.Diagnostics.TraceSource Nuget package because those classes are already part of netstandard2.0.

It is disconcerting that Visual Studio did the wrong thing when you added a reference to this Nuget package (which we *do* need when targettings netstandard1.6).